src/overlays/actors/ovl_En_Egblock/z_en_egblock.h

```diff
@@ -2,16 +2,44 @@
 #define Z_EN_EGBLOCK_H
 
 #include "global.h"
+// #include "overlays/actors/ovl_En_Egol/z_en_egol.h"
 
 struct EnEgblock;
 
 typedef void (*EnEgblockActionFunc)(struct EnEgblock*, PlayState*);
 
+#define GET_EGBLOCK_PARAM_7F(thisx) ((thisx)->params & 0x7F)
+#define GET_EGBLOCK_PARAM_F80(thisx) (((thisx)->params >> 7) & 0x1F)
+#define GET_EGBLOCK_TYPE(thisx) (((thisx)->params >> 12) & 0xF)
```

Comment:
I personally prefer `ACTORNAME_GET_WHATEVER` vs. `GET_ACTORNAME_WHATEVER`, and I think that's what I do in my actors too. But I don't think we've settled on a common standard for this.